src/course-search/tests/FacetListBase.test.jsx

```diff
@@ -125,4 +137,25 @@ describe('<FacetListBase />', () => {
     // assert page was deleted and subjects were not
     expect(history.location.search).toEqual('?showAll=1&subjects=Communication');
   });
+
+  test('renders a searchable dropdown', async () => {
+    renderWithRouter(<SearchData><FacetListBase {...searchableDropdownProps} /></SearchData>);
+
+    // assert the "no options" message does not show
+    expect(screen.queryByText(NO_OPTIONS_FOUND)).not.toBeInTheDocument();
+
+    await act(async () => {
+      fireEvent.click(screen.queryByText('Skills'));
+    });
+    expect(screen.queryByPlaceholderText('Find a skill')).toBeInTheDocument();
+    expect(screen.queryByText('Blockchain')).toBeInTheDocument();
+    expect(screen.queryByText('Cryptocurrency')).toBeInTheDocument();
+
+    await act(async () => {
```

Comment:
This seems like a test that is making sure that `searchForItems` is properly called, which is great, but it should be a new test, not part of this one.